app/controllers/admin/partners_controller.rb

```
@@ -29,7 +33,7 @@ def destroy
   private
 
   def get_partner
-    @partner = Partner.find(params[:id])
+    @partner = Partner.find_by(id: params[:id])
```

You should to use `find` because it will raise the error if NOT FOUND, `find_by` will return `nil`